docs/concepts-broker-load-balancing-types.md

```diff
@@ -0,0 +1,63 @@
+---
+id: concepts-broker-load-balancing-types
+title: Types
+sidebar_label: "Types"
+---
+
+Pulsar provides three types of broker load balancers:
+
+- Simple broker load balancer: it existed since Pulsar's inception, but now is deprecated. No one uses it.
+
+- [Modular broker load balancer](#side-by-side-comparisons)
+
+- [Extensible broker load balancer](#side-by-side-comparisons)
+
+The modular and extensible broker load balancer use similar implementation logic and algorithm and have similar functionalities. For example, they both employ a similar approach to distributing data loads among brokers, including:
+
+- Dynamic [bundle-broker assignment](./concepts-broker-load-balancing-concepts.md#bundle-assignment)
+
+- Dynamic [bundle splitting](./concepts-broker-load-balancing-concepts.md#bundle-splitting)
+
+- Dynamic [bundle unloading (shedding)](./concepts-broker-load-balancing-concepts.md#bundle-unloading)  
+
+As a result, brokers dynamically collect and share the current load with other brokers for better load balancing decisions. 
+
+## Side-by-side comparisons
+
+The extensible broker load balancer refactors the logic of the modular broker load balancer with better modularity and is less dependent on ZooKeeper, which makes Pulsar more stable and scalable. 
+
+Dimension|Modular broker load balancer|Extensible broker load balancer
+|---|---|---
+Available version|Pulsar 1.7.0 <br/> <br/>2017|Pulsar 3.0.0 <br/> <br/>2023
+Architecture|ZooKeeper plays an important role, which may cause a large overhead of load data replication.<br/> <br/> All broker load data and all bundle load data are stored in ZooKeeper and replicated to all brokers via ZooKeeper watchers. | ZooKeeper is less dependent (only lookup data is stored in ZooKeeper), which reduces the overhead of load data replication and minimizes data processing delays.<br/> <br/> Pulsar reads and writes data via system topics and table view rather than ZooKeeper: <br/> <br/> - [System topics](./concepts-messaging.md#system-topic): global ownership data is stored in bundle state channels within persistent topics. <br/> <br/> - [Table view](./concepts-clients.md#tableview): global ownership data is replicated using the compacted table view, where 1) All broker load data is replicated to all brokers via non-persistent topics. 2) Only the top-n bundle load data from each broker is replicated to the leader brokers via non-persistent topics. <br/> <br/> The absolute size of the shared load data and the complexity of the replication decreased due to passing the data through memory-only, non-persistent topics. 
```

Comment:
```suggestion
Architecture|ZooKeeper plays an important role, which may cause a large overhead of load data replication.<br/> <br/> All broker-loaded data and all bundle-loaded data are stored in ZooKeeper and replicated to all brokers via ZooKeeper watchers. | ZooKeeper is less dependent (only lookup data is stored in ZooKeeper), which reduces the overhead of load data replication and minimizes data processing delays.<br/> <br/> Pulsar reads and writes data via system topics and table view rather than ZooKeeper: <br/> <br/> - [System topics](./concepts-messaging.md#system-topic): global ownership data is stored in bundle state channels within persistent topics. <br/> <br/> - [Table view](./concepts-clients.md#tableview): global ownership data is replicated using the compacted table view, where 1) All broker-loaded data is replicated to all brokers via non-persistent topics. 2) Only the top-n bundle-loaded data from each broker is replicated to the leader brokers via non-persistent topics. <br/> <br/> The absolute size of the shared load data and the complexity of the replication decreased due to passing the data through memory-only, non-persistent topics. 
```